docs/svelte-testing-library/setup.md

```diff
@@ -47,25 +47,36 @@ with any testing framework and runner you're comfortable with.
     }
     ```
 
-5.  This is optional but it is recommended, you can install
+5.  If you are using es6 modules in your project you have to add jest's babel
```

Comment:
would be nicer to say "ES6" and "Jest" instead of "es6" and "jest"

Reply:
Agreed and done 🙂